src/main/resources/db/changelog/changes/01-create-books-table.yaml

```
@@ -0,0 +1,44 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: Roma1990Melnichuk
+      changes:
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: title
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: author
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: isbn
+                  type: VARCHAR(13)
+                  constraints:
+                    unique: true
+                    nullable: false
+              - column:
+                  name: price
+                  type: DECIMAL(10,2)
```

```suggestion
                  type: DECIMAL(10,2)
                  constraints:
                    nullable: false
```
use the same constraint that the entity class